Scenes/Objects/ghost.gd

```diff
@@ -0,0 +1,72 @@
+extends Enemy
+class_name Ghost
+
+const AGGROTIME: float = 2.0
+const SPEED : float = 45.0
+var TP_TIMER: float = 10
+
+var target: Node2D
+var aggroTimer: float
+
+var idleTimer: float = 0
+var hasFinishFollowIntoRoom: bool = false
```

Comment:
I made a statemachine and state class to inherit from and this is what happens 😭

Reply:
lol